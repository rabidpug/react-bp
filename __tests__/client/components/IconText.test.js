import IconText from 'Components/IconText';
import React from 'react';
import { shallow, } from 'enzyme';

describe(
  'IconText', () => {
    it(
      'Should render the IconText', () => {
        const wrapper = shallow( <IconText /> );

        expect( wrapper ).toMatchSnapshot();
      }
    );
  }
);
