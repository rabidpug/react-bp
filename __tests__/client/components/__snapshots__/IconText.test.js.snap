// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IconText Should render the IconText 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <IconText />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <Icon
          style={
            Object {
              "marginRight": 8,
            }
          }
          type={undefined}
        />,
        undefined,
      ],
      "onClick": undefined,
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "style": Object {
            "marginRight": 8,
          },
          "type": undefined,
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      undefined,
    ],
    "type": "span",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Icon
            style={
              Object {
                "marginRight": 8,
              }
            }
            type={undefined}
          />,
          undefined,
        ],
        "onClick": undefined,
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "style": Object {
              "marginRight": 8,
            },
            "type": undefined,
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        undefined,
      ],
      "type": "span",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
